dev/breeze/src/airflow_breeze/utils/docs_publisher.py

```diff
@@ -58,15 +58,15 @@ def _build_dir(self) -> str:
     @property
     def _current_version(self):
         if not self.is_versioned:
-            raise Exception("This documentation package is not versioned")
+            raise ValueError("This documentation package is not versioned")
```

Comment:
RuntimeError.

Additional description: "Make sure to add version in provider.yaml for the package".